### ISSUE TITLE
export default is missing

### ISSUE THREAD

**Issue by adrianjost** [vuejs/vue-test-utils#1415]
### Version
1.0.0-beta.31

### Reproduction link
[https://jsfiddle.net/q7d08jwy/](https://jsfiddle.net/q7d08jwy/)

### Steps to reproduce
switch any import statement of the test utils from:
`import * as utils from "@vue/test-utils";`
to:
`import utils from "@vue/test-utils";`

### What is expected?
testUtils should contain all the exports like it did in v1.0.0-beta-28 so you can do 
import testUtils from "@vue/test-utils";
testUtils.mount(...)

### What is actually happening?
testUtils is undefined

---
I think we shouldn't restrict options.

The changes where introduced in this commit:
https://github.com/vuejs/vue-test-utils/commit/d45e248ea9d0b9665ba19a020483db493219b481

This is definetly a breaking change and should be documented in the release notes if intended!

**Comment by dobromir-hristov** [comment]
Hey there. 
The old way of exporting was technically a hack. It was default exporting an object, which was then being transpiled to a CJS file, which you could both destructure import as well as importing entirely. This however was causing issues with the development process of the library, as we are now using the raw, untranspiled src to run tests against.

Most people should have already switched to named imports, before v 28, that is why we did not list it in the changelog.

Sorry for the inconvenience.

**Comment by adrianjost** [comment]
I understand, but as far as I can see it is also possible to have both solution at the same time and it's also fairly easy to implement.
You just need to store the object you are currently exporting and export it both as the default export as well as the named exports.

```
const methods = {
  // ...
}
export methods
export default methods
```
This should not break anything or is there anything I do not understand yet? If not I would be happy to open up a pull request if you want.

**Comment by lmiller1990** [comment]
I think if we can have both solutions at the same time that would be ideal. From what @dobromir-hristov said, it was change since it was causing issues with the development process.

If you are able to get it to work, and also run `yarn test:unit:only:dev` and it still works, I think it's fine to make a PR!

Edit: looks like adding `export default` https://github.com/vuejs/vue-test-utils/blob/9dc7d1a648b168f740154ead593e3f1dbd9a52f7/packages/test-utils/src/index.js#L32 here works. You will need to update here https://github.com/vuejs/vue-test-utils/blob/dev/packages/server-test-utils/src/index.js as well.

**Comment by lbogdan** [comment]
I don't think it makes sense to go against best practices (export the main thing as the default export and additional things as named exports) only to keep backwards compatibility with a few projects, and also:
- this is still in beta, so API changes are to be expected;
- there's [a single place in the documentation](https://vue-test-utils.vuejs.org/guides/usage-without-a-build-step-node.html#requiring-the-libraries) that uses `const VueTestUtils = require('@vue/test-utils')` (and no `import VueTestUtils from '@vue/test-utils'`), for projects that don't use `webpack`, which I'd estimate is a small percentage of real-word, production projects.

**Comment by lbogdan** [comment]
Turns out that because of the CJS - ESM interop `const VueTestUtils = require('@vue/test-utils')` still works, so the only one not working is `import VueTestUtils from '@vue/test-utils'`, which, I reiterate, is not referenced anywhere in the docs.

**Comment by lmiller1990** [comment]
Ok, I think the above points are compelling enough to not add this. Sorry for the inconvenience @adrianjost - we should have documented this more clearly in our release notes.

Since there is a pretty simple fix available, I'm going to close this issue.

**Comment by adrianjost** [comment]
I totally understand that this software is still in beta and changes may happen. I was just wondering because I had found this issue: https://github.com/vuejs/vue-test-utils/issues/873 where it was implicitly implemented and it was in the docs a couple of versions earlier. I am 100% fine leaving it as is now because of the easy fix. 

---

**FYI:** I tried implementing export default again and this simple (but ugly duplicate) code seem to work fine (`yarn test:unit:only:dev` and `yarn build` are working without any problem). I have had some problems with flow (which I never used before) when putting the object into a variable thats why the code is that ugly. But it does work.

[I did the adjustments in this file](https://github.com/vuejs/vue-test-utils/blob/dev/packages/test-utils/src/index.js)
```js
export default {
  createLocalVue,
  createWrapper,
  config,
  enableAutoDestroy,
  mount,
  resetAutoDestroyState,
  shallow,
  shallowMount,
  RouterLinkStub,
  Wrapper,
  WrapperArray
};
export {
  createLocalVue,
  createWrapper,
  config,
  enableAutoDestroy,
  mount,
  resetAutoDestroyState,
  shallow,
  shallowMount,
  RouterLinkStub,
  Wrapper,
  WrapperArray,
}
```
